bom/deployment/pom.xml

```diff
@@ -37,7 +37,7 @@
             <!-- Camel -->
             <dependency>
                 <groupId>org.apache.camel.quarkus</groupId>
-                <artifactId>camel-quarkus-parent</artifactId>
+                <artifactId>camel-quarkus-bom</artifactId>
```

Comment:
This is a deployment BOM. So we should rather import `camel-quarkus-bom-deployment`. There is a fix in #49, BTW.

Reply:
You would know best, I just used whatever worked :). I updated the PR.